### ISSUE TITLE
VirtualMachineInfrastructureProviders API endpoint is missing some field

### ISSUE THREAD

**Issue by baelen-git** [CiscoDevNet/terraform-provider-intersight#25]
The API spec is missing some fields for this endpoint.
https://intersight.com/apidocs/apirefs/kubernetes/VirtualMachineInfrastructureProviders/model/

Under InfraConfig there should be:
Datacenter
Cluster
Passphrase
ResourcePool

When I create this Infra Provider via the Intersight GUI I find that it sends the following body:
{
    "Name": "test",
    "Organization": {
        "ObjectType": "organization.Organization",
        "Moid": "xxxxx"
    },
    "Target": "xxxxx",
    "InfraConfig": {
        "ObjectType": "kubernetes.EsxiVirtualMachineInfraConfig",
        "Cluster": "HX-MGMT",
        "Datastore": "ALL_FLASH",
        "Interfaces": [
            "vm-network-99"
        ],
        "Passphrase": "DITISMIJNPW",
        "ResourcePool": "resource_pool_default"
    }
}

when I try to specify these fields from Terraform it get the following:

Error: Unsupported argument

  on iks.tf line 81, in resource "intersight_kubernetes_virtual_machine_infrastructure_provider" "AMSLAB-All-Flash":
  81:         datacenter = "ALL_FLASH" 

An argument named "datacenter" is not expected here.

**Comment by code-lucidal58** [comment]
The openapi specification used for 1.0.9-3144 did have datastore and cluster properties. In the upcoming release provider 0.1.4, the provider will be updated as per 1.0.9-3181 version of openapi specification. It will have the properties that are missing currently. The release is due in a day.

**Comment by baelen-git** [comment]
Hi,

We discovered that those properties where available under the additional_properties property.
It is possible right now to provide them though the provider in this way:
```
       additional_properties = jsonencode({
            Datastore = var.Datastore
            Cluster = var.Cluster
            Passphrase = var.Passphrase
            ResourcePool = var.ResourcePool
        })
```

I noticed that many properties in the Intersight provider are 'hidden' under the additional_property field.
The difficulty is that we have no way of knowing which properties are available under the additional_property field.
Because this problem is for many resources in the intersight terraform provider I will create a separate issue for this.

Thanks for your assist with the InfraProvider.
